warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/EventMapper.java

```diff
@@ -463,14 +481,22 @@ public void map(K1 key, V1 value, Context context) throws IOException, Interrupt
             value.setAuxProperty(ErrorDataTypeHandler.PROCESSED_COUNT, "1");
         }
 
-        // Determine whether the event date is greater than the interval. Excluding fatal error events.
+        // Determine whether the event date is earlier than the configured interval. Excluding fatal error events.
         if (!value.fatalError() && null != myInterval && 0L != myInterval && (value.getDate() < (now.get() - myInterval))) {
             if (log.isInfoEnabled())
                 log.info("Event with time " + value.getDate() + " older than specified interval of " + (now.get() - myInterval) + ", skipping...");
             getCounter(context, IngestInput.OLD_EVENT).increment(1);
             return;
         }
 
+        // Determine whether the event date is too far in the future compared to the configured interval. Excluding fatal error events.
+        if (!value.fatalError() && null != myFutureInterval && 0L != myFutureInterval && (value.getDate() < (now.get() - myFutureInterval))) {
```

Comment:
This is a future interval, so the check should whether the value.getDate() is greater than now plus the future interval.